challenges/objects-arrays.js

```diff
@@ -61,35 +100,53 @@ The resulting contact information strings should have a space between the first
 
 Log the result of your new array. */
 const contactInfo = [];
+for (let i = 0; i < graduates.length; i++) {
+	let studentInfo = graduates[first_name];
+	constactInfo.push(studentInfo.email);
+}
 console.log(contactInfo);
```

Comment:
While you are close, this code does not run. Can you spot why? Line 104 is what is breaking the code. -2pts